C++/source/searching/jump_search.cpp

```diff
@@ -0,0 +1,54 @@
+// C++ program to implement Jump Search
+ 
+#include <bits/stdc++.h>
```

Comment:
This does not work for all compilers. Include only those headers that are required.